.github/workflows/publish-to-demo.yaml

```diff
@@ -0,0 +1,91 @@
+name: "Verify & Publish to Demo"
+
+on:
+  workflow_dispatch:
+    inputs:
+      refType:
+        type: choice
+        description: "Find branch name, commit SHA, or tag?"
+        options:
+          - Branch name
+          - Commit SHA
+          - Tag
+        default: Branch name
+      gitRef:
+        description: "Input branch name, commit SHA, or tag"
+        required: true
+        type: string
+        default: main
+
```

Comment:
thinking about this - we probably also want to deploy this when there are changes to the 'main' branch, so we should put in:
```
push:
    branches:
      - main 
```
 as well